deploy/scripts/src/common/requestGenerator/txmaReqGen.ts

```diff
@@ -0,0 +1,389 @@
+    event_timestamp_ms: Math.floor(Date.now()),
+    user: {
+      user_id: userID,
+      govuk_signin_journey_id: journeyID,
+      ip_address: '1.2.3.4',
+      email: emailID,
+      session_id: uuidv4(),
+      persistent_session_id: uuidv4(),
+      transaction_id: uuidv4()
+    }
+  }
+}
+
+export function generateAuthAuthorizationInitiated(journeyID: string): AuthAuthorizationInitiated {
+  return {
+    client_id: 'performanceTestClientId',
+    component_id: 'perfTest',
+    event_name: 'AUTH_AUTHORIZATION_INITIATED',
+    event_timestamp_ms: Math.floor(Date.now()),
+    extensions: {
+      'client-name': 'PerfTest',
+      new_authentication_required: false
+    },
+    timestamp: Math.floor(Date.now() / 1000),
+    user: {
+      govuk_signin_journey_id: journeyID,
+      ip_address: '1.2.3.4',
+      persistent_session_id: uuidv4(),
+      session_id: uuidv4()
+    }
+  }
+}
+
+export function generateAuthCodeVerified(emailID: string, journeyID: string, userID: string): AuthCodeVerified {
+  return {
+    client_id: 'performanceTestClientId',
+    component_id: 'perfTest',
+    event_name: 'AUTH_CODE_VERIFIED',
+    event_timestamp_ms: Math.floor(Date.now()),
+    extension: {
+      MFACodeEntered: '123',
+      'account-recovery': false,
+      'journey-type': 'SIGN_IN',
+      loginFailureCount: 0,
+      'mfa-type': 'SMS',
+      'notification-type': 'SMS'
+    },
+    timestamp: Math.floor(Date.now() / 1000),
+    user: {
+      email: emailID,
+      govuk_sigin_journey_id: journeyID,
+      ip_address: '1.2.3.4',
+      persistent_session_id: uuidv4(),
+      session_id: uuidv4(),
+      user_id: userID
+    }
+  }
+}
+
+export function generateAuthUpdatePhone(
+  emailID: string,
+  journeyID: string,
+  userID: string
+): AuthUpdateProfilePhoneNumber {
+  return {
+    client_id: 'performanceTestClientId',
+    component_id: 'perfTest',
+    event_name: 'AUTH_UPDATE_PROFILE_PHONE_NUMBER',
+    event_timestamp_ms: Math.floor(Date.now()),
+    extensions: {
+      'account-recovery': false,
+      'mfa-type': 'SMS',
+      phone_number_country_code: 44
+    },
+    timestamp: Math.floor(Date.now() / 1000),
+    user: {
+      email: emailID,
+      govuk_signin_journey_id: journeyID,
+      ip_address: '1.2.3.4',
+      persistent_session_id: uuidv4(),
+      phone: '07123456789',
+      session_id: uuidv4(),
+      user_id: userID
+    }
+  }
+}
+
+export function generateIPVJourneyStart(journeyID: string, userID: string): IPVJourneyStart {
+  return {
+    client_id: 'performanceTestClientId',
+    component_id: 'perfTest',
+    event_name: 'IPV_JOURNEY_START',
+    event_timestamp_ms: Math.floor(Date.now()),
+    extensions: {
+      reprove_identity: false,
+      vtr: 'CI'
+    },
+    restricted: {
+      device_infomation: {
+        encoded: 'RW5jb2RlZCBkYXRhIHdpbGwgYmUgaGVyZQ==' //Device information retrieved from cloudfront headers encoded in Base6 (have currently made up base64).
```

Comment:
Add the below comment at the end of the line. This should tell the pre-commit hook that this is not a secret.
// pragma: allowlist secret